### ISSUE TITLE
java.lang.IllegalArgumentException: JAVA_HOME not defined

### ISSUE THREAD

**Issue by mokkun** [cbeust/kobalt#27]
### Steps
1. Ensure that `JAVA_HOME` is not defined.
2. [Download Kobalt](https://github.com/cbeust/kobalt/releases/latest) and unzip the file (`$KOBALT_HOME`).
3. Change to a existing Java project directory and execute `$KOBALT_HOME/kobaltw --init`.

### Actual
The following error message is displayed.
```
Exception in thread "main" java.lang.ExceptionInInitializerError
	at com.beust.kobalt.SystemProperties$Companion.<clinit>(SystemProperties.kt:7)
	at com.beust.kobalt.BuildScriptKt.homeDir(BuildScript.kt:13)
	at com.beust.kobalt.misc.KFiles.<clinit>(KFiles.kt:44)
	at com.beust.kobalt.misc.KFiles$$FastClassByGuice$$fc521753.newInstance(<generated>)
	at com.google.inject.internal.cglib.reflect.$FastConstructor.newInstance(FastConstructor.java:40)
	at com.google.inject.internal.DefaultConstructionProxyFactory$1.newInstance(DefaultConstructionProxyFactory.java:61)
	at com.google.inject.internal.ConstructorInjector.provision(ConstructorInjector.java:105)
	at com.google.inject.internal.ConstructorInjector.construct(ConstructorInjector.java:85)
	at com.google.inject.internal.ConstructorBindingImpl$Factory.get(ConstructorBindingImpl.java:267)
	at com.google.inject.internal.SingleParameterInjector.inject(SingleParameterInjector.java:38)
	at com.google.inject.internal.SingleParameterInjector.getAll(SingleParameterInjector.java:62)
	at com.google.inject.internal.ConstructorInjector.provision(ConstructorInjector.java:104)
	at com.google.inject.internal.ConstructorInjector.construct(ConstructorInjector.java:85)
	at com.google.inject.internal.ConstructorBindingImpl$Factory.get(ConstructorBindingImpl.java:267)
	at com.google.inject.internal.ProviderToInternalFactoryAdapter$1.call(ProviderToInternalFactoryAdapter.java:46)
	at com.google.inject.internal.InjectorImpl.callInContext(InjectorImpl.java:1103)
	at com.google.inject.internal.ProviderToInternalFactoryAdapter.get(ProviderToInternalFactoryAdapter.java:40)
	at com.google.inject.internal.SingletonScope$1.get(SingletonScope.java:145)
	at com.google.inject.internal.InternalFactoryToProviderAdapter.get(InternalFactoryToProviderAdapter.java:41)
	at com.google.inject.internal.SingleParameterInjector.inject(SingleParameterInjector.java:38)
	at com.google.inject.internal.SingleParameterInjector.getAll(SingleParameterInjector.java:62)
	at com.google.inject.internal.ConstructorInjector.provision(ConstructorInjector.java:104)
	at com.google.inject.internal.ConstructorInjector.construct(ConstructorInjector.java:85)
	at com.google.inject.internal.ConstructorBindingImpl$Factory.get(ConstructorBindingImpl.java:267)
	at com.google.inject.internal.InjectorImpl$2$1.call(InjectorImpl.java:1016)
	at com.google.inject.internal.InjectorImpl.callInContext(InjectorImpl.java:1092)
	at com.google.inject.internal.InjectorImpl$2.get(InjectorImpl.java:1012)
	at com.google.inject.internal.InjectorImpl.getInstance(InjectorImpl.java:1051)
	at com.beust.kobalt.MainKt.mainNoExit(Main.kt:40)
	at com.beust.kobalt.MainKt.main(Main.kt:23)
Caused by: java.lang.IllegalArgumentException: JAVA_HOME not defined
	at com.beust.kobalt.SystemProperties.<clinit>(SystemProperties.kt:8)
	... 30 more
```

### Expected
No error messages or to mention in the install guide the need of defining `JAVA_HOME`.

### Environment
Operational System
`OSX 10.11.1 (15B42)`

Java
```
java version "1.7.0_79"
Java(TM) SE Runtime Environment (build 1.7.0_79-b15)
Java HotSpot(TM) 64-Bit Server VM (build 24.79-b02, mixed mode)
```

**Comment by cbeust** [comment]
I should be able to get rid of the requirement for $JAVA_HOME eventually.

**Comment by mokkun** [comment]
@cbeust looks like it was fixed with version `0.219`. I've reproduced the steps and all went fine.
Feel free to close this issue if you agree.

**Comment by cbeust** [comment]
Yup, thanks for the reminder!
